pages/samvera/manager_guide/hyrax_3.x/user-collections-3.x.md

```diff
@@ -0,0 +1,143 @@
+
+## Adding Works to Collections
+The visibility of a collection does not affect the visibility of a work. For example, if a Public work is added to a Private collection, the work will remain Public, but a link to the collection itself can not be discovered. Additionally, if the user does not have permission to view a restricted work in a shared collection, they will not be able to see the work in the collection edit page.
+
+There are several ways to add works to collections. Note that all of these options require the collection to have already been created. 
+
+### Adding Works to Collections from the Dashboard
+1. From the Dashboard, click"Works"
+2. Select one or more works
+3. Click the "Add to Collection" button
+4. Select the collection in the dialog box, then click "Save Changes"
+
+![screenshot of works page with two items selected. The add to collection button is displayed above](\images\screenshots\add_to_collection.PNG)
+
+### Adding Works to Collections in Browse view
+
+1. On a work page, click "Edit" 
+2. Select the "Relationships" tab
+3. Search for the collection in the "Add to collection" box
+4. Click "Add"
+
+![detail of relationships tab on the work page, showing collections section with search box and add button](\images\screenshots\collections_browse.PNG)
+
+### Adding New Works to a Collection
+
+1. When creating a new work, click the "Relationships" tab 
+2. Search for the collection in the "Add to collection" box
+3. Click "Add"
+
+## Editing Collections
+### Edit Collection Description
+
+1. Click on the "Collections" link in the Dashboard
+2. Click on the collection 
+3. Click the "Edit collection" button
+
+![detail of collections edit page, showing title and description text boxes](\images\screenshots\collection_edit.PNG)
+
+### Edit Collection Branding
+Users have the option to add a custom banner or logo to their collections. 
+
+1. Click on the "Collections" link in the Dashboard 
+2. Click on the collection 
+3. Click the "Edit Collection" button 
+4. Select the "Branding" tab at the top of the page 
+
+![detail of collections branding, showing banner and logo upload sections](\images\screenshots\collection_branding.PNG)
+
+### Edit Collection Visibility
+
+1. Click on the "Collections" link in the Dashboard 
+2. Click on the collection 
+3. Click the "Edit Collection" button 
+4. Select the "Discovery" tab at the top of the page 
+
+Collections can be either public, limited to institution only, or private.
+
+![detail of collections visibility, showing public, institution and private visibility selections. Private is selected](\images\screenshots\collection_visibility.PNG)
+
+### Sharing Collections with Other Users
+1. Click on the "Collections" link in the Dashboard
+2. Click on the collection 
+3. Click the "Edit Collection" button. 
+4. Select the "Sharing" tab at the top of the page
+
+![detail of collections sharing, showing add group and add user search boxes](\images\screenshots\collection_sharing.PNG)
```

Comment:
@rjkati : I merged this PR quickly, but during re-examination I noticed an error and missing file.  Do you have this image?